compute/computer_setup/install_magaox_rpi.rst

```diff
@@ -0,0 +1,103 @@
+Installing MagAO-X on a Raspberry Pi
+====================================
+
+
+Configuring the Raspberry Pi
+----------------------------
+
+Prepare the MicroSD card
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first step is to download the Raspberry Pi Imager from the `Raspberry Pi Website <https://www.raspberrypi.com/software/>`__. The MicroSD card can then 
```

Comment:
This is a windows specific thing (or it might exist on Mac too).  So you could re-word this to be more generic like "Using a program to image an SD card (on windows...)